### ISSUE TITLE
Args in set_exception_handler() are need to be changed in php7.0?

### ISSUE THREAD

**Issue by Keech** [phalcon/incubator#656]
I noticed that `set_exception_handler()` is probably needed to change.
Now your code in `Phalcon\Error\Handler` is below.

```
set_exception_handler(function (\Exception $e) {});
```

But PHP reference is saying, 
>If the handler needs to work on both PHP 5 and 7, you should remove the type declaration from the handler, while code that is being migrated to work on PHP 7 exclusively can simply replace the Exception type declaration with Throwable instead.
(http://php.net/manual/en/migration70.incompatible.php)

So, I think you need to fix like this, though how about it?

```
set_exception_handler(function (\Throwable $e) {});
```

Thnaks.

**Comment by sergeyklay** [comment]
Fixed in the 3.0.x branch https://github.com/phalcon/incubator/pull/655/files

**Comment by Keech** [comment]
@sergeyklay Thanks. I was using older one. When updated it, then solved.
